server.js

```diff
@@ -59,6 +61,30 @@ app.get('/api', function apiIndex(req, res) {
   })
 });
 
+app.get('/api/profile', function apiProfile(req, res) {
```

Comment:
You do all the work to put your profile information in the database, but still use this hardcoded object. You don't need all of that profile DB code.